hooks/image/70-flatpak-manifest

```diff
@@ -56,25 +56,10 @@ if os.environ.get('EIB_FLATPAK_ENABLE', 'false') != 'true':
     write_manifest(data, hook_group)
     exit(0)
 
-# Open the flatpak installation in the OS /var
+# Open the flatpak installation in the OS /var/lib/flatpak or split disk
+# /var/endless-extra/flatpak.
```

Comment:
Are you planning on dropping the split-disk stuff in a separate MR? aiui we’re dropping that feature too.

Reply:
Yeah, separate feature to delete. There's a lot more to the split disk step than just this.

Reply:
SGTM. Merge when the mood takes you

Reply:
The mood has struck me. 👯